crates/orchestrator/api.toml

```diff
@@ -32,11 +32,13 @@ Get the latest temporary node index only for generating validator's key pair for
 
 # POST the node's node index to generate public key for pubkey collection
```

Comment:
Let's remove `to generate public key` if it's easy to update, but feel free to leave it to later PR.